src/Classical/linear_code.jl

```diff
@@ -396,6 +396,19 @@ end
 #############################
      # general functions
 #############################
+function information_set(C::AbstractLinearCode)
+    """
+    Returns the indexs of the pivot columns and an information set which 
+    is the submatrix of G given by these columns.
+    """
+    if C.G == C.G_stand
+        pivot_inds=[x for x ∈ 1:C.n]
+    else
+        nonpivot_inds = _non_pivot_cols(C.G, :nsp) 
```

Comment:
So... there's a bug in a separate algorithm and it mainly calls this function. It's probably worth reading this over and seeing if it's actually correct haha. Maybe we should unit test this...

Reply:
so _non_pivot_cols currently assumes input is in rref form. I would like to rename it to _rref_non_pivot_cols